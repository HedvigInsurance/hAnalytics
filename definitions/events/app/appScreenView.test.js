const validateEvent = require("commons/validateEvent.js");

test("validate", () => {
  const event = validateEvent(__filename, {
    screenName: "home",
  });
  expect(event).toMatchSnapshot();
});
