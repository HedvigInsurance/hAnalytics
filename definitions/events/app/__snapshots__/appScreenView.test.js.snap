// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate 1`] = `
Object {
  "name": "app_screen_view",
  "variables": Object {
    "screen_name": undefined,
  },
}
`;
